specs/eip4844/polynomial-commitments.md

```diff
@@ -222,44 +223,34 @@ def blob_to_polynomial(blob: Blob) -> Polynomial:
     return polynomial
 ```
 
-#### `compute_challenges`
+#### `compute_challenge`
 
 ```python
-def compute_challenges(polynomials: Sequence[Polynomial],
-                       commitments: Sequence[KZGCommitment]) -> Tuple[Sequence[BLSFieldElement], BLSFieldElement]:
+def compute_challenge(polynomial: Polynomial,
```

Comment:
There is argument miscmatch here. All the callers of `compute_challenge()` pass a blob as the first arg.

Reply:
Yep. Fixed.